mod_ci/controllers.py

```diff
@@ -463,23 +463,45 @@ def queue_test(db, gh_commit, commit, test_type, branch="master", pr_nr=0):
     """
     from run import log
 
-    fork = Fork.query.filter(Fork.github.like("%/CCExtractor/ccextractor.git")).first()
+    fork = Fork.query.filter(Fork.github.like(
+        "%/CCExtractor/ccextractor.git")).first()
+    fork_id = Test.query.filter(Test.fork_id == Fork.id).first()
```

Comment:
Did you test this? As far as I can see, this will return the first instance of a Test that satisfied a condition that won't work either, not a fork id.